### PR TITLE
:memo: Documentation: Add Snippets for Maven and Gradle

### PR DESCRIPTION
**Pull request by rharriso**
## Description

Documentation: I thought it was confusing that the the fuel coroutines 'download' link was pointing to the Android package. This change adds Snippets for installing with Maven and Gradle package managers.

### CODE REVIEW

**Comment by SleeplessByte** [comment]
It doesn't point to the "android" package. Inspect the files: https://bintray.com/kittinunf/maven/Fuel-Android/2.2.1#files/com/github/kittinunf/fuel.

On `bintray` it's _called_ Fuel-Android. A misnomer.

**Comment by rharriso** [comment]
Ah ok. That is a little confusing.
I used the link on jcenter to get the POM entry for just `fuel-coroutines`.

File: fuel-coroutines/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ The coroutines extension package for [`Fuel`](../README.md).
 
 ## Installation
 
-You can [download](https://bintray.com/kittinunf/maven/Fuel-Android/_latestVersion) and install `fuel-coroutines` with `Maven` and `Gradle`. The coroutines package has the following dependencies:
+You can [download](https://mvnrepository.com/artifact/com.github.kittinunf.fuel/fuel-coroutines/latest) and install `fuel-coroutines` with `Maven` and `Gradle`. The coroutines package has the following dependencies:
```
Comment (kittinunf) [inline fuel-coroutines/README.md:6]: 🤔 What do you think about this link, instead? https://bintray.com/kittinunf/maven/Fuel-Android/2.2.1#files/com%2Fgithub%2Fkittinunf%2Ffuel%2Ffuel-coroutines

Comment (rharriso) [reply fuel-coroutines/README.md]: doesn't seem to work any better, that link doesn't go right to the files. Takes you to the same landing page.

**Comment by kittinunf** [comment]
@rharriso Really sorry for the naming! Truth to be told this is being started as an Android project. Then, the more it grow, we (our contributors) extracted the core part out from the project. The name was just a legacy misnomer. 😭

**Comment by rharriso** [comment]
@kittinunf no worries, I think that's pretty common for Kotlin projects. Perhad we should add an entry for what that POM or Gradle import should look like? That why I added the link the jcenter. I wasn't able to find it directly from this documentation.

**Comment by kittinunf** [comment]
@rharriso Okay, good idea. Would you mind help us doing so?

**Comment by rharriso** [comment]
@kittinunf Can do, I'll try to get to it in the next week.

**Comment by rharriso** [comment]
@kittinunf I've updated the Markdown file, adding snippets for maven and gradle installation.

**Pull request merged**